src/docs/glossary.md

```diff
@@ -4,68 +4,94 @@ eleventyNavigation:
   key: Glossary
 ---
 
-# Glossary and Buzzwords
+# Glossary
+
+This page provides two lists of terms — one for [Eleventy-specific terminology](#eleventy-specific-terminology) that may be useful for when building out a project using Eleventy, and one for [industry jargon](#industry-terms-and-buzzwords) that may be useful for understanding context.
+
+_Work in progress: {% include "edit-on-github.njk" %}_
+
+## Eleventy-Specific Terminology
+
+### Template
+
+A content file written in a [format such as Markdown, HTML, Liquid, Nunjucks, and more](/docs/languages/), which Eleventy transforms into a page (or pages) in the built site. Templates can access [data](#data) exposed through the [data cascade](#data-cascade) with templating syntax.
+
+### Layout
+
+A template which wraps around another template, typically to provide the scaffolding markup for content to sit in.
+
+[Read more about using layouts.](/docs/layouts/)
+
+### Data
+
+Variables that can be used inside [templates](#template) and [layouts](#layout) using templating syntax. The data for a given template is aggregated through a process called the [data cascade](#data-cascade).
```

Comment:
Possibly more clear? Data doesn't have to lead to variables, it can feed pagination as well (maybe that can be worked in?)

```suggestion
Exposed via variables that can be used inside [templates](#template) and [layouts](#layout) using templating syntax. The data for a given template is aggregated through a process called the [data cascade](#data-cascade).
```